src/platform/packages/shared/presentation/presentation_publishing/interfaces/fetch/publishes_reload.ts

```diff
@@ -10,7 +10,7 @@
 import type { Observable } from 'rxjs';
 
 export interface PublishesReload {
-  reload$: Omit<Observable<void>, 'next'>;
+  reload$: Observable<void>;
```

Comment:
Why does this PR remove the Omit?

Reply:
Observables don't have a `next` function. They're readonly.